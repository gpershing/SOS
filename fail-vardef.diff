diff: tests/fail-vardef.err: No such file or directory
