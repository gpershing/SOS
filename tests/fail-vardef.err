Fatal error: exception Failure("Could not resolve type id notatype")
