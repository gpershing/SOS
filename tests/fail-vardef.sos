//defining a variable with an undefined type id

x: notatype = 10
