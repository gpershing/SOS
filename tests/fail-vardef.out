Could not resolve type id x
